template/update_layout.tmpl

```diff
@@ -2,6 +2,25 @@
 {{ $project_name_alphanumeric_underscore := (regexp `-`).ReplaceAllString ((regexp `[^A-Za-z0-9_-]`).ReplaceAllString .input_project_name ``) `_` -}}
 {{ $root_dir := .input_root_dir}}
 
+{{ if (eq .input_setup_cicd_and_project `Project_Only`) }}
+    {{ skip (printf `%s/%s` $root_dir `.azure`) }}
+    {{ skip (printf `%s/%s` $root_dir `.github`) }}
+    {{ skip (printf `%s/%s` $root_dir `docs`) }}
+    {{ skip (printf `%s/%s` $root_dir `.gitignore`) }}
+    {{ skip (printf `%s/%s` $root_dir `README.md`) }}
+    {{ skip (printf `%s/%s` $root_dir `test-requirements.txt`) }}
+    {{ skip (printf `%s/%s` $root_dir `cicd.tar.gz`) }}
+    {{ skip (printf `%s/%s` $root_dir `cicd_params.json`) }}
+{{ else if (eq .input_setup_cicd_and_project `CICD_Only`) }}
+    {{ skip (printf `%s/%s` $root_dir $project_name_alphanumeric_underscore) }}
+    {{ skip (printf `%s/%s/%s-%s` $root_dir `.github/workflows` .input_project_name `run-tests.yml`) }}
```

Comment:
We're not skipping the entire directory though, because we still need the top-level workflows to be generated like the deploy CI/CD workflow

Reply:
Do you know if it's possible for us to define a regex like `input_project_name-*`? If so that would make our life a ton easier and we wouldn't have to specify each file cc @shreyas-goenka

Reply:
@arpitjasa-db Yeah we support glob patterns for the file name, you don't need to pass in all file paths!

You can see https://pkg.go.dev/path#Match for the glob pattern syntax supported.

Reply:
Perfect ty! Updated